lib/opentok/opentok.rb

```diff
@@ -36,6 +36,7 @@ module OpenTok
   #     the token.
   #
   #   @param [Hash] options A hash defining options for the token.
+  #   @option options [String] :token_type The type of token to generate. Must be one of 'T1' or 'JWT'.
```

Comment:
@jeffswartz I think we'll need to update the OT docs [here](https://tokbox.com/developer/sdks/ruby/) and [here](https://tokbox.com/developer/guides/create-token/ruby/) to cover this new param.